pom.xml

```diff
@@ -424,9 +425,14 @@
 
             <!--JAKARTA-->
             <dependency>
-                <groupId>com.sun.mail</groupId>
-                <artifactId>jakarta.mail</artifactId>
-                <version>${com.sun.jakarta.mail.version}</version>
+                <groupId>jakarta.mail</groupId>
+                <artifactId>jakarta.mail-api</artifactId>
+                <version>${jakarta.mail.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.eclipse.angus</groupId>
+                <artifactId>angus-mail</artifactId>
+                <version>${angus.mail.version}</version>
             </dependency>
```

Comment:
If I understand correctly, this dependency will be used just in tests, correct?

And just a silly question but why do we need it at all now? It is a replacement for what dependency?

Reply:
Thanks, Vasek, for the review. I've just rechecked the dependencies and made some minor changes.